.env.example

```diff
@@ -46,7 +46,8 @@ HEROKU_STAGING            # (used in db/seeds.rb)
 MAILGUN_API_KEY=app-account-key
 MAILGUN_DOMAIN=app-email-domain
 
-#SHF_SEED_USERS=4         # The number of users to create during seeding (Takes precedence of the default number in seeds.db). Seed with a minimum of 4 users to cover: admin, no application, single application, double application
+SHF_SEED_USERS=100               # The number of users to create during seeding (Takes precedence of the default number in seeds.db). Seed with a minimum of 4 users to cover: admin, no application, single application, double application
+SHF_FIRST_MEMBERSHIP_NUMBER=100  # The first membershipnumber used if the database is recreated.
```

Comment:
Make want to make clear that this is the first number used _in the dev DB_.  (that is, has nothing to do with production).

Reply:
Interesting. My thinking was that it will also be used in the production environment if the system is set up anew. That's why I only mentioned the `db_recreate` as criterium. But I understand your point: there is a difference between "a system in production" and the "production environment".  And it should be made clear that this key has no bearing on the system in production.